File: app/src/main/java/com/lifedawn/bestweather/retrofit/responses/flickr/Comments.java

package com.lifedawn.bestweather.retrofit.responses.flickr;

import java.io.Serializable;
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Comments implements Serializable
{

    @SerializedName("_content")
    @Expose
    private Integer content;
    private final static long serialVersionUID = -751650815179992814L;

    public Integer getContent() {
        return content;
    }

    public void setContent(Integer content) {
        this.content = content;
    }

}

File: app/src/main/java/com/lifedawn/bestweather/data/remote/retrofit/responses/flickr/Comments.kt
package com.lifedawn.bestweather.data.remote.retrofit.responses.flickr

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName
import java.io.Serializable

class Comments : Serializable {
    @SerializedName("_content") @Expose var content: Int? = null

    companion object {
        private const val serialVersionUID = -751650815179992814L
    }
}